openmoc/plotter.py

```diff
@@ -179,13 +182,13 @@ def plot_segments(track_generator):
   plt.xlim([x.min(), x.max()])
   plt.ylim([y.min(), y.max()])
 
-  title = 'Segments for ' + str(num_azim) + ' angles and ' + str(spacing) + \
-        ' cm spacing'
+  title = 'Segments for ' + str(num_azim) + ' angles, ' + str(spacing) + \
+        ' cm spacing, and ' + str(z[0]) + ' z-level'
```

Comment:
Perhaps the z-level could be a [subtitle](http://matplotlib.sourcearchive.com/documentation/0.98.3/classmatplotlib_1_1figure_1_1Figure_c5a0bfb9075702f39846bad94aee5463.html)?